lib/common.sh

```diff
@@ -2,6 +2,12 @@
 
 [[ ":$PATH:" != *":/usr/local/go/bin:"* ]] && PATH="$PATH:/usr/local/go/bin"
 
+# Verify that passwordless sudo is configured correctly
+if ! sudo -l | grep -q '(ALL) NOPASSWD: ALL';then
```

Comment:
We need `sudo -nl`, otherwise this prompt will hang if the user doesn't have passwordless sudo permissions.

Reply:
Thanks, fixed.